Automattic/wp-calypso
# Focused Launch: PlanDetails View

#### Changes proposed in this Pull Request

* Implements the detailed plans step in Focused Launch

#### Testing instructions

1. In the project root, run `yarn && yarn start`. In a separate terminal window, run `cd apps/editing-toolkit && yarn dev --sync`.
2. Sandbox YOUR_SITE.wordpress.com.
3. Visit https://YOUR_SITE.wordpress.com/wp-admin/post-new.php.
4. In console, type `wp.data.dispatch('automattic/launch' ).openFocusedLaunch()`.
5. Click "View all plans". 
7. You should see the plans grid exactly as in the Figma file.

Fixes https://github.com/Automattic/wp-calypso/issues/46867
